content/en/aws/kms/index.md

```diff
@@ -0,0 +1,123 @@
+
+{{< command >}}
+$ awslocal kms describe-key --key-id 010a4301-4205-4df8-ae52-4c2895d47326
+{{</ command >}}
+
+We can use the key now to encrypt something. Let's say, `some important stuff`:
+
+{{< command >}}
+$ awslocal kms encrypt \
+      --key-id 010a4301-4205-4df8-ae52-4c2895d47326 \
+      --plaintext "some important stuff" \
+      --output text \
+      --query CiphertextBlob \
+  | base64 --decode > my_encrypted_data
+{{</ command >}}
+
+Why do we require such a complicated command? The `Encrypt` Operation itself looks like this:
+
+{{< command >}}
+$ awslocal kms encrypt \
+      --key-id 010a4301-4205-4df8-ae52-4c2895d47326 \
+      --plaintext "some important stuff"
+{{</ command >}}
+
+And its output might be something like
+
+```sh
+{
+    "CiphertextBlob": "MDA4NDk4ZDYtMGU1ZS00NjU3LWEyZTQtMDliYTcwNDgyMmJkxw1eaLVUeUZA7bZzp+k7VAAAAAAAAAAAAAAAAAAAAAAetG9/5Znpw83/4xhwObc6Fc2B73y1ZvIigwahKopT0Q==",
+    "KeyId": "arn:aws:kms:us-east-1:000000000000:key/010a4301-4205-4df8-ae52-4c2895d47326",
+    "EncryptionAlgorithm": "SYMMETRIC_DEFAULT"
+}
+```
+
+The value of the `CiphertextBlob` field is a Base64-encoded result of encryption of our plaintext. In order to use KMS to decrypt this, we have to get rid of this Base64-encoding first. So to our `Encrypt` operation we add the `output` parameter, requesting the output to be in the text form instead of the default JSON. We also supply the `query` parameter, so the output only contains the value of the specified `CiphertextBlob` field. Then we feed the result to `base64` tool, asking it to remove the Base64 encoding, producing a binary output. We save that output to a file for future use. **Note:** Careful with the `CiphertextBlob` value for the `query` parameter - the AWS CLI doesn't check if the requested field is even if the output, so if you have a typo, the command would succeed, but the data sent to `base64` is going to be just a word `None`. Which is going to lead to unexpected results later. 
+
+We can then decrypt the content of our file using the same KMS key:
+
+{{< command >}}
+$ awslocal kms decrypt \
+      --ciphertext-blob fileb://my_encrypted_data \
+      --output text \
+      --query Plaintext
+  | base64 --decode
+{{</ command >}}
+
+Here we ask KMS to decrypt contents of the binary file created during our previous `Encrypt` call. We do not have to specify the `key-id` while decrypting our file - when encrypting something with a symmetric key, AWS includes the key ID into the encrypted data, so can figure out itself which key to use for decryption. With asymmetric keys the `key-id` has to be specified, though.
+
+As with the previous `Encrypt` call, we have to get rid of the Base64-encoding to actually get our data, so we use the `output` and `query` parameters along with the `base64` tool here as well. If everything went fine, the output is going to be
+
+```sh
+some important stuff
```

Comment:
Please give atleast a sample output for the end user to verify if they followed the example correctly

Reply:
```sh
some important stuff
```
**is** the output of the command above.